src/Compilers/Core/Portable/MetadataReader/MetadataDecoder.cs

```diff
@@ -717,24 +717,32 @@ private TypeSymbol GetTypeOfTypeDef(TypeDefinitionHandle typeDef, out bool isNoP
                 {
                     var isAllowed = false;
 
-                    switch (allowedRequiredModifierType)
+                    if ((allowedRequiredModifierType & AllowedRequiredModifierType.System_Runtime_InteropServices_InAttribute) != 0 &&
```

Comment:
>if ((allowedRequiredModifierType & AllowedRequiredModifierType.System_Runtime_InteropServices_InAttribute) != 0 && [](start = 20, length = 114)

It looks like these changes are going to conflict with changes made in [https://github.com/dotnet/roslyn/pull/43355](https://github.com/dotnet/roslyn/pull/43355). Consider coordinating with @333fred to make changes in both PRs in a manner that would minimize the churn duing merge

Reply:
Fred, this change allows for looking for more than one modreq at a certain position. Would you mind taking this change into your branch to minimize merge conflicts?

---
In reply to: [410990682](https://github.com/dotnet/roslyn/pull/43275#discussion_r410990682) [](ancestors = 410990682)